wordpress-mobile/WordPress-Android
# Feature/initial quick start prompt cherrypicked

This PR is a copy of reversed #8014 that for some reason pulled two other pull requests with it when it was merged. I reversed all the changes and recreated this PR. It's already approved so I'm just going to merge it.

Description is the original PR:

Part of #7707

This PR adds functionality for showing Quick Start dialog snack bars (with icons) after the user picks tutorial from Quick Start screen.

To test:

Navigate to Quick Start activity from My Site fragment.
Click on each tutorial (Create Site is completed by default), and make sure that the Quick Start dialog snack bar with relevant text appears at MySite fragment.
Snackbar will not be recreated when orientation changes while it's still visible.
